### ISSUE TITLE
Unable to use plugin with Serverless Framework v3

### ISSUE THREAD

**Issue by AndriiSushkoTEG** [yndlingsfar/serverless-openapi-integration-helper#13]
Plugin can't  be installed with version 3 of framework
```npm ERR! Could not resolve dependency:
npm ERR! peer serverless@"2.x" from serverless-openapi-integration-helper@1.8.0
```
Looks like `peerDependencies` need to be updated

**Comment by alexbezugliy** [comment]
End up with the same problem recently - the plugin is working fine on sls3. Can you guys update peerDependencies to refer serverless@3.x?

**Comment by yndlingsfar** [comment]
solved :-) Use Version 2.0

**Comment by yndlingsfar** [comment]
Published a new version
